x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/EnterpriseSearch.java

```diff
@@ -71,8 +71,11 @@
 import java.util.function.Supplier;
 
 public class EnterpriseSearch extends Plugin implements ActionPlugin, SystemIndexPlugin {
-    public static final String ENGINE_API_ENDPOINT = "_engine";
-    public static final String BEHAVIORAL_ANALYTICS_API_ENDPOINT = "_behavioral_analytics";
+    public static final String APPLICATION_API_ENDPOINT = "_application";
+
+    public static final String SEARCH_APPLICATION_API_ENDPOINT = "_application/search_application";
```

Comment:
nitpick: could we define this as 
`public static final String SEARCH_APPLICATION_API_ENDPOINT = APPLICATION_API_ENDPOINT + "/search_application";` (the same pattern as `BEHAVIORAL_ANALYTICS_API_ENDPOINT`)